web-share/resources/manual-helper.js

```diff
@@ -50,6 +50,11 @@ function setupManualShareTest(expected_share_data) {
   item = document.createElement('li');
   list.appendChild(item);
   item.innerText = `url = "${url}"`;
+  if (!!files) {
+    item = document.createElement('li');
+    list.appendChild(item);
+    item.innerText = `files = ${files.length} file(s)`;
```

Comment:
Worth showing the content of each? I guess just knowing you got them might be enough.

Reply:
Now showing content.